### PR TITLE
Release 7.4.0. Locale Fix PR-320; Undocumented Parameters PR-317; Wor…

### PR DESCRIPTION
**Pull request by tmconnors**
…ker Policy for TaskRouter PR-313

### CODE REVIEW

File: src/main/java/com/twilio/rest/chat/v1/service/channel/Message.java [hunk 1]
```diff
@@ -96,14 +96,12 @@ public static MessageDeleter deleter(final String serviceSid,
...
-     * @param body The body
      * @return MessageUpdater capable of executing the update
      */
     public static MessageUpdater updater(final String serviceSid, 
                                          final String channelSid, 
-                                         final String sid, 
-                                         final String body) {
-        return new MessageUpdater(serviceSid, channelSid, sid, body);
+                                         final String sid) {
+        return new MessageUpdater(serviceSid, channelSid, sid);
```
Comment (tmconnors) [inline src/main/java/com/twilio/rest/chat/v1/service/channel/Message.java]: This is backwards incompatible.

File: src/main/java/com/twilio/rest/taskrouter/v1/workspace/WorkspaceStatisticsFetcher.java [hunk 2]
```diff
@@ -16,12 +17,13 @@
...
 import com.twilio.http.TwilioRestClient;
 import com.twilio.rest.Domains;
+import org.joda.time.DateTime;
 
 public class WorkspaceStatisticsFetcher extends Fetcher<WorkspaceStatistics> {
     private final String workspaceSid;
     private Integer minutes;
-    private String startDate;
-    private String endDate;
+    private DateTime startDate;
```
Comment (tmconnors) [inline src/main/java/com/twilio/rest/taskrouter/v1/workspace/WorkspaceStatisticsFetcher.java]: This is backwards incompatible.

File: src/main/java/com/twilio/rest/taskrouter/v1/workspace/Task.java [hunk 3]
```diff
@@ -121,14 +121,10 @@ public static TaskReader reader(final String workspaceSid) {
...
      * @param workspaceSid The workspace_sid
-     * @param attributes The attributes
-     * @param workflowSid The workflow_sid
      * @return TaskCreator capable of executing the create
      */
-    public static TaskCreator creator(final String workspaceSid, 
-                                      final String attributes, 
-                                      final String workflowSid) {
-        return new TaskCreator(workspaceSid, attributes, workflowSid);
+    public static TaskCreator creator(final String workspaceSid) {
```
Comment (tmconnors) [inline src/main/java/com/twilio/rest/taskrouter/v1/workspace/Task.java]: This is backwards incompatible.

File: src/main/java/com/twilio/rest/taskrouter/v1/workspace/taskqueue/TaskQueueStatistics.java [hunk 4]
```diff
@@ -85,27 +86,31 @@ public static TaskQueueStatistics fromJson(final InputStream json, final ObjectM
...
     private final String taskQueueSid;
     private final String workspaceSid;
+    private final URI url;
 
     @JsonCreator
     private TaskQueueStatistics(@JsonProperty("account_sid")
                                 final String accountSid, 
                                 @JsonProperty("cumulative")
-                                final Map<String, String> cumulative, 
+                                final Map<String, Object> cumulative,
```
Comment (tmconnors) [inline src/main/java/com/twilio/rest/taskrouter/v1/workspace/taskqueue/TaskQueueStatistics.java]: This is backwards incompatible.

File: src/main/java/com/twilio/rest/taskrouter/v1/workspace/taskqueue/TaskQueueStatisticsFetcher.java [hunk 5]
```diff
@@ -51,17 +50,6 @@ public TaskQueueStatisticsFetcher setEndDate(final DateTime endDate) {
     }
 
     /**
-     * The friendly_name.
-     * 
-     * @param friendlyName The friendly_name
-     * @return this
-     */
-    public TaskQueueStatisticsFetcher setFriendlyName(final String friendlyName) {
```
Comment (tmconnors) [inline src/main/java/com/twilio/rest/taskrouter/v1/workspace/taskqueue/TaskQueueStatisticsFetcher.java]: This is backwards incompatible.

File: src/main/java/com/twilio/rest/taskrouter/v1/workspace/taskqueue/TaskQueuesStatistics.java [hunk 6]
```diff
@@ -83,7 +83,7 @@ public static TaskQueuesStatistics fromJson(final InputStream json, final Object
     }
 
     private final String accountSid;
-    private final Map<String, String> cumulative;
+    private final Map<String, Object> cumulative;
```
Comment (tmconnors) [inline src/main/java/com/twilio/rest/taskrouter/v1/workspace/taskqueue/TaskQueuesStatistics.java]: This is backwards incompatible.

**Comment by efossier** [comment]
+1

**Pull request merged**